widgets/src/lib/interfaces/Libraries.ts

```diff
@@ -101,6 +101,13 @@ model = Speech2Text.from_pretrained(
 speech, rate = soundfile.read("speech.wav")
 text, *_ = model(speech)`;
 
+const espnetAudioClassification = (model: ModelData) =>
```

Comment:
this has to be `const speechbrainAudioClassification`, unless speechbrain is related to espnet?

Reply:
Done, I messed working between two libraries at same time.